# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../commonc++/gcc44.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

cidr.cpp:205: error: invalid conversion from 'const char*' to 'char*'
cidr.cpp:335: error: invalid conversion from 'const char*' to 'char*'

--- commoncpp2-1.7.3/src/cidr.cpp.vanilla	2010-02-06 11:06:29.000000000 +0100
+++ commoncpp2-1.7.3/src/cidr.cpp	2010-02-06 11:07:40.000000000 +0100
@@ -199,11 +199,7 @@
 	bitset((bit_t *)&netmask, getMask(cp));
 	setString(cbuf, sizeof(cbuf), cp);
 	
-#if defined(_MSC_VER) && _MSC_VER >= 1500
 	ep = (char *)strchr(cp, '/');
-#else
- 	ep = strchr(cp, '/');
-#endif
 
 	if(ep)
 		*ep = 0;
@@ -332,7 +328,7 @@
 	memset(&netmask, 0, sizeof(netmask));
 	bitset((bit_t *)&netmask, getMask(cp));
 	setString(cbuf, sizeof(cbuf), cp);
-	ep = strchr(cp, '/');
+	ep = (char*)strchr(cp, '/');
 	if(ep)
 		*ep = 0;
 
